Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/cli/blobs/CliBlob.java

```diff
@@ -27,8 +27,8 @@
  * <p>
  * Paraphrasing from ISO 23271:2012 11.24.2.4 (p272):
  * - If the first one byte of the 'blob' is 0bbbbbbb_2: size is bbbbbbb_2 bytes.
- * - If the first two bytes are 10bbbbbb_2 and x: size is (bbbbbb_2 << 8 + x) bytes.
- * - If the first four bytes are 110bbbbb_2, x, y, and z: size is (bbbbb_2<<24 + x<<16 + y<<8 + z) bytes.
+ * - If the first two bytes are 10bbbbbb_2 and x: size is (bbbbbb_2 &lt;&lt; 8 + x) bytes.
+ * - If the first four bytes are 110bbbbb_2, x, y, and z: size is (bbbbb_2&lt;&lt;24 + x&lt;&lt;16 + y&lt;&lt;8 + z) bytes.
```

Comment:
Here is an example where @literal might be more appropriate.  This becomes a bit crazy looking when trying to read the javadoc from source.  Most people can deal with one &amp;lt;, but this isn't too useful too look at in this form.